### PR TITLE
Good heavens, it's that time again. Brig Remap 2.0

### PR DESCRIPTION
**Pull request by FreeStylaLT**
It is that time again, I just can't leave this department alone.
**Comparison: 
(Click for Increased Size)**

Current Brig:
img

New Brig:
img

**Main Objectives of this remap:**

- Expand the brig to better suit current pop

Pretty much everyone who plays Sec has told me that it can barely, if at all, support higher pop rounds, especially processing and evidence storage. They'd get cramped and overfilled.
- Get rid of the unnecessary vertical design of past brig

The sec pod pilot hallway along with the stacked armories were the core issues I have had. They were unnecessary and felt like they were _forced_ to be there because there was no other place for HoS / Pod Pilot.
- Streamline the flow of the department

Going from the last point, the result of it was that some people had to go through that hallway / stacked armories in order to get from point A to B, especially as HoS / Pod Pilot, as their respective huts were placed inside that crappy hallway. I changed that to be entirely horizontal.

- Make each cell equally able to be escaped from / visited

The tucked away / isolated brig cells were always an issue, as they both prevented them from being visited and escaped from easily. This is where my new lobby comes in, which increases the visibility Sec would have of most cells, while allowing them all to be visitable (or be aided in escape) equally.

**Why a whole remap?** 
Because the current brig has an integral, central block that the rest of the brig has been built around from. That's the Warden's office + armories. Moving or changing them would result in changes everywhere, while them being the literal cause of the verticality issues the brig has. A full remap was the only option if you wanted to fix these issues, as they encompassed the whole brig.

**Notable differences:**

New Warden / Sub-Armory / Armory Complex:
img
img
The core of the department, this type of change would not have been adequately possible to implement in the past brig without a complete remap. It makes both armories equally accessible to the Warden, with the main armory being directly accessible to the rest of Sec. 

I believe this is a direct improvement to what we have - as it removes the forced verticality of the department, while improving accessibility.

Space side has been left mostly untouched - still a tucked away armory under permabrig, though it is smaller.

Shutter at work:
img
Shutter has Warden and above access.

New Evidence Storage:
img
Roomier, more "secure", still exposed to maint.

New Gamma Armory:
img
I shrunk it, moved the entrance to the left, this resulted in it being able to fit in much more neatly into the brig proper, as opposed to being attached to it.

New Detective's Office:
img
I shrunk it. (Big surprise.) It no longer has a forensic hut, or direct maint access. It now border space.

New Processing:
img
Bigger. (Actual surprise) That's about it. Gets privacy shutters to shield from yelling cellmates:
Shutters in action:
img
Shutter is Officer and above access
They do not block the side door.

New Lobby:
img
Much bigger, or rather taller. This is because the cells, as you can see, are aligned along the walls of the lobby, instead of there being separate cell blocks tucked away in an edge of brig.

New Execution Room / Firing Range:
img
Execution room has been made fancier and directly linked to the firing range. "Decency" shutters have been added, along with the ability to space / gas / gas AND space / gas AND space WHILE electrocuting the poor soul.

_The space shutters have electrified grilles under them._

Shutters in action:
img

Maint:
img
No longer expands past brig, entrance aligns with the rest of the wall (for better or worse).
The atmos closet can now be used to isolate brig's air supply. The external airlock has been made horizontal. Processing, instead of Detective, gets direct access to Maint now. **That Maint/Processing airlock has a lockdown shutter, so it gets locked down along with the rest of brig.**

Overall, the pipes from IAA maint and up have been _vastly_ improved, the sizes of rooms that were deemed too cramped / small were increased by ~1 row / column of tiles and rooms that were too large have been optimized, like Firing Range becoming a compact, single row instead of the two large ones.

Otherwise, I tried to keep everything else in roughly the same locations and arranged in a similar way. The end result is something that looks like the old 2015 brig, just.. not as terrible, hopefully.

All piping  is courtesy of FalseIncarnate!

### CODE REVIEW

**Comment by Xantholne** [comment]
As a security main I support a new brig whole heartingly.

**Comment by KasparoVy** [comment]
This actually looks fantastic.

**Comment by Lady-Luck** [comment]
I am very much against the bathroom area. It is definitely going to lead to officers giving people swirlies, I guarantee it.

**Comment by SamHPurp** [comment]
@Lady-Luck irrelevant, there was already one in perma.

Also, whats a little swirlie amongst friends?

**Comment by Lady-Luck** [comment]
>irrelevant, there was already one in perma.

Not irrelevant. Perma is quite the trip, and it's, well, in perma.

**Comment by FreeStylaLT** [comment]
@SamHPurp Nukie shuttle is fine and far away from Perma in both North and Northeast locations.

**Comment by SamHPurp** [comment]
@FreeStylaLT thanks for checking. 👍

**Comment by Lady-Luck** [comment]
There should probably be paper in the meeting room.

**Comment by Lady-Luck** [comment]
Could interrogation be 'flipped' so there can be a door going from processing directly into interrogation?

**Comment by Anoonki** [comment]
I love the new look of this.
The Warden's gonna be a *LOT* busier now, though. Which I'm not sure *arguably* the actively busiest guy on the station needed, running between the two cell blocks to stop the fuckler smashing them up.

**Comment by Fruerlund** [comment]
Yes yes yes, cant wait to try it out. Fantastic work doing it and great job asking for advice/inputs and critism.

**Comment by Xantholne** [comment]
NOW WE CAN GAS, SPACE, AND ELECTROCUTE THE BASTARD RUINING OUR WORKPLACE

**Comment by AndriiYukhymchak** [comment]
I support it, but move ablative armor set to the middle and leave shotguns on the north wall, like we have now. This will ensure "steal ablative armor" will be as difficult as it was before

**Comment by sothangel** [comment]
Looks neat, is that little 1x3 slither of space between the armoury and the floor flashers viewable by cameras?

**Comment by FreeStylaLT** [comment]
@sothangel Yes, it is.

**Comment by theColdflame** [comment]
I don't play this server so I won't comment on my overall thoughts but here's some criticism and possible issues to consider:

spacing is not a sanctioned execution method on paradise, nor is it actually hugely practical given the current lethality of space- it's also needlessly painful and inhumane from an IC standpoint. electric chair is more practical imo

can't tell if it does by the screenshot but perma shouldn't have banana seeds

number of cells is 6 (I think?) which is probably too few if it's planned to be use with a pop of 120+ players on an LRP server, admittedly I've not played recently but I feel expecting 5% of the crew at the absolute maximum to be shitters is optimistic

already a problem in the current brig but worse here- sec don't have anywhere to store a large inflow of prisoners  (as old temp detainment used to serve the purpose of)- this gives sec an extremely hard time during conversion gamrmodes after winning a teamfight or after busting a riot

lobby is extremely vulnerable to being blocked off and there's only one side entrance now (also easily blocked off)- seems like it might be intentional though. 

the removal of maint access to the right is a heavy nerf to sec, it gives them no discreet options for navigation, slows their response time, and gives them no escape option during direct brig attacks. also makes the map less interesting to navigate since you have to retrace your steps an backtrack a lot now as sec

**Comment by tigercat2000** [comment]
@theColdflame It's not for spacing prisoners (although that's probably how it'll be used), it's a Gas Chamber like the one on MetaStation. The blast doors are for clearing out the gasses.

**Comment by tigercat2000** [comment]
@FreeStylaLT A full resolution render of the changes, you could put this in the OP if you want

**Comment by DaveTheHeadcrab** [comment]
@FreeStylaLT The Gamma armory borders the Warden's armory.. But opens into the prison block. Might be more practical to switch which side that door is on and have it open into the armory.

**Comment by FreeStylaLT** [comment]
@DaveTheHeadcrab That's how it was in previous iterations, until I decided that having it be easily accessible to every officer was a more practical solution, especially considering the usual dire situations that Gamma is used in, where a Warden / HoS may not be available, cus dead. 

If Gamma was accessible from Warden's Secondary Armory only, an officer would have to bust through at least one electrified window to get to it, which is wholly impractical, while the only real benefit of that would be the fact that Warden / HoS would be more in control of the distribution of the equipment, which I am not sure of the benefits of, either, considering the situations that currently warrant Gamma (massive threats that have gone out of control, in most cases)

**Comment by lolexploit** [comment]
Why is the bathroom area without reinforced walls facing space?

**Comment by SamHPurp** [comment]
@lolexploit gotta give antags a chance somehow. Some areas in the current map don't have reinforced.

**Comment by Spartan6** [comment]
@SamHPurp As a rule of thumb, all walls facing space should be reinforced other than the odd example.

**Comment by SamHPurp** [comment]
@Spartan6 As an example, the Firing Range wasn't.

**Comment by warior4356** [comment]
I would like to see a door airlock into the brig itself, adding doors as i have drawn here to makr it much harder to run into the brig when greytiding. Actual antags lack a reason to rush in.
brig edit

**Comment by Anticept** [comment]
Eyeballing the shit out of it @FreeStylaLT . Here's what I'm seeing.

That air alarm should be moved out of permabrig and next to the APC. There's no point to it being inside, and it needs better access for setting contaminated to clear N2O.

Is solitary part of the hallway area? It lacks an APC (just like before) or air alarm (this is different from before).

The electric chair doesn't use power from cables, it uses power from the APC by setting a brief surge of 5000w to equipment. That wire is not needed. That said, it looks like you actually meant to run it to the APC just below the chair, so this needs fixed.

There is a wire that is not curved, south entrance of the labor shuttle, southmost set of windows. At the intersection. This may or may not electrify the glass, and should be fixed to ensure it does. 
image

Does the new bathroom get an APC and air alarm? Or is it part of another area?

Is the security locker are supposed to have an APC?

You have a main line feed going under the window near the vendor. Please move that to pass under a door. 
image

That atmos change over there near the escape shuttle is hilarious. I love it. It's so delightfully evil. However, still quite ineffective unless the way vents work is changed (they will never pump till there is enough of a pressure drop).

This wire is ugly as sin: image

There's no vent or scrubber on the north side of the interrogation room.

You oopsied here, south east side of brig lobby cells: 
image That waste pipe manifold needs to be moved up one.

Where are the windoors for each of the brig cells in the lobby? Are they covered up by the blast doors?

This air alarm is pointless and should be moved to some other place in the room image

Does this area need an APC? 
image . It looks like you meant to put one in (curved wire that goes nowhere in the middle).

**Comment by Citinited** [comment]
It makes me kind of sad that antagonists can't break into the gamma armoury from space now, but I guess that makes sense.

I feel like the HoS' office should have windows put between it and the detective's office, instead of walls.

Detective should also maybe get a maintenance door?

Great changes though! :+1:

**Comment by Anticept** [comment]
New screenie, @FreeStylaLT so I can look at it again?

**Comment by FreeStylaLT** [comment]
@Anticept OP was updated with a new full brig image.

**Comment by Lady-Luck** [comment]
I'd very much like to see interrogation changed in the following manner:



It'd make it easy to take a prisoner into interrogation straight from processing.

**Comment by Anticept** [comment]
@FreeStylaLT i'll look at more later, but that long hallway up to perma is still missing an apc.

**Comment by FalseIncarnate** [comment]
@Anticept I think I can clear up a few of your points:

1. The permabrig air alarm location:
This actually is how it is currently, meant to be to report the air status of the actual permabrig. Sec officers can't access the air alarm anyways, and an atmos tech can simply set the scrubbers from their control computer when the alarm goes active, or don some internals and do it in person. Additionally, the giant stationary scrubbers are controlled by a computer right next to where you pump the N2O in anyways, and those clear EVERYTHING that isn't O2 and N2 rather effectively.

8. The atmos closet:
Again, this is actually how it used to be, though more recent maint changes actually made it so you'd have to re-pipe a bit of the supply line there to be able to actually isolate it as intended. Further, the purpose of the ability to isolate security is so that if there happens to be some sort of major atmos shenanigans in the supply loop, you can isolate the brig to be safe from the issue and run them from a back-up canister until the loop can be cleared, though you can just as easily slip in a tank of N2O and knock out all of security so you can pursue your more traitorous intentions with less red-shirt resistance. If you wanted to do that though, you'd want to mess with their air alarms to either decrease the local pressure and then refill, or up the output pressure to throw more gas into the room, things easily achieved by someone with the means to hook up a canister of their favorite atmos gas(es) in that closet anyways.

9. No vent/scrubber in interrogation observation:
Intended, since it is such a small room which is easily re-pressurized/de-pressurized by opening the hall door and probably won't see much use anyways. The vents in the main area are because it (theoretically) will have more people in it for longer and provides an additional escape spot for those with the means to utilize it (changelings, atmos traitors, that pesky diona nymph that keeps biting you) as well as an extra entry point for xenos and that pesky diona nymph that likes to stalk you and chirp.

10. Bad scrubber manifold placement:
Whoops, that was a mistake, somehow that one slipped past the checks I ran or was mistakenly edited afterwords.

12. Bad air alarm placement:
That definitely is a stupid spot, though that is also how it is in the current version. Neither myself nor Free touched that alarm (to my knowledge), so neither of us paid attention to it. Could be swapped with the intercom on the tile above it to correct.

As for the suggestion made by @Lady-Luck 
Originally Free had it so that you would access interrogation from processing, but I convinced him to change that to the current layout for a number of reasons. First, you generally have the doors to interrogation and observation both off the same hallway. This makes it easier for an officer to leave interrogation and enter observation quickly and without breaking the illusion that the suspect is alone and un-monitored by the officer who is in reality on the other side of the one-way window. Second, this layout also makes it so it is slightly easier for someone to be broken OUT of interrogation by a cohort, since the sec officer has to go further to enter maintenance where you are breaching. Third, having interrogation separated from processing by the jaunt through the hallway makes the suspect feel more intimidated, since they are being moved out of view of other suspects who might have been brought in with them. That extra bit of isolation can sometimes be the difference between defiance and compliance since the suspect will feel that things are more serious and that potential allies and aid are further than they really are.

**Comment by FalseIncarnate** [comment]
The atmos piping changes extend from the EVA maint junction by the clown and mime offices to the bar maint door, and everything north of there. This includes cryostorage, the bathrooms, Mr. Changs, the barber, the IAA and magistrate offices, all of the security maint tunnel, and the fitness room (dorms).

Except for 2 universal pipe adapters in the atmos closet, all pipes pass under doorways, plastic flaps, or windows in the affected areas. No longer will you have to cut down walls to get at those pipes. Disposal pipes in the new brig area likewise only pass under doors and such (this is especially noticeable by Mr. Changs, which used to be an utter horror of disposal piping under walls).

**Comment by Anticept** [comment]
@FalseIncarnate actually, for small rooms, why don't we use passive vents more often? Seems like a really simple way to do it than running around with a bunch of scrubbers and vents.

**Comment by Citinited** [comment]
Passive vents would make it really easy for contaminants to pass between rooms, and make it incredibly easy for someone to over-pressurise rooms, though.

:+1:

**Comment by Anticept** [comment]
I don't see how passive vents make it any different for someone to overpressurize a room with the resources we already have.

As for contaminants: indeed, but it's for small rooms connected to larger ones with scrubbers and all that :P

**Comment by Citinited** [comment]
If you're using passive vents then the vent will attempt to equalise the pipe pressure with the pressure on the tile the vent is on.

So if you pressurise the pipes to 500kPa, the room will eventually equalise at that pressure.

**Comment by Citinited** [comment]
Fair enough, seems I mistunderstood you.

**Comment by FreeStylaLT** [comment]
OP Updated with a new full picture.

**Comment by SamHPurp** [comment]
Can we not just take this opportunity to remove the ball-gag?

I don't know where you'd put it, but the light switch in the detective's office feels hidden and out of the way. Think of how light switches are everywhere else, in real life etc... near the doors.

**Comment by TravellingMerchant** [comment]
One thing I dont like is if something causes ALL the glass to break. Either an explosion or vampire or something right in the middle there, that makes all the cells unusable.

**Comment by taukausanake** [comment]
@TravellingMerchant Well the cells do still have electrified grills under them. Would they be unusable still?  And don't grills cast ThunderBolt if you throw things at them now? Lastly, engineering always needs more work

**Comment by Anticept** [comment]
Not all of my concerns have been addressed yet. In particular, that hallway is still missing an APC.

**Comment by FreeStylaLT** [comment]
@Anticept 
>Is the security locker are supposed to have an APC?
No, it is a joint area with the briefing room, titled "Security Office". It is like this in current Brig, am unsure of changing it to be two separate ones.

>Does the new bathroom get an APC and air alarm? Or is it part of another area?
At the moment - no, it is a separate, "Vacant Office" area, I will be adding alarms to it in the next commit.

>Is solitary part of the hallway area? It lacks an APC (just like before) or air alarm (this is different from before).
Yes, it is part of the hallway and Perma, area titled "Prison Wing". The air alarm was pointless to have there, as it was already accessible from the Prison Wing hallway.

>Does this area need an APC? It looks like you meant to put one in (curved wire that goes nowhere in the middle). [Prisong Wing hallway]
No. The APC is located inside Permabrig proper, just like previous brig.

**Comment by Anticept** [comment]
Now I know what the brig is missing.

Fire extinguishers.

**Comment by Spartan6** [comment]
@Anticept That's because if a criminal gets free of his cuffs and starts running round, he can grab a fire extinguisher and start battering a skull in.

**Comment by Citinited** [comment]
God forbid security get robusted by their inmates if they get free!

In any case, if a prisoner gets free what exactly will they be able to do? Security can tase / pepperspray / flashbang them easily, can hear windows getting hit if the inmate tries to break out of the brig, etc. Secure your prisoners better.

**Comment by Jovaniph** [comment]
Why not have the box of prisoner ID inside the Prisoner Locker room?

**Comment by mafemergency** [comment]
> @Anticept That's because if a criminal gets free of his cuffs and starts running round, he can grab a fire extinguisher and start battering a skull in.

Make a childproofed extinguisher made out of plastic, nerfing it in some way (volume or pressure) so it can safely bounce off heads.

Something I'd also like to see is a medpack in perma because when things get wild the prisoners are unlikely to receive any attention unless there's a prison break.

**Comment by taukausanake** [comment]
> Make a childproofed extinguisher made out of plastic, nerfing it in some way (volume or pressure) so it can safely bounce off heads.

Isn't that what the portable extinguisher is?

**Comment by Anticept** [comment]
I would prefer fire extinguishers be in the main halls. If a prisoner manages to escape to the halls, sec deserves every bit of being beaten in the head with a fire extinguisher.

**Comment by deep-fried-faceplant** [comment]
Remove the windows from processing. People standing in the hallway and adjacent rooms will be able to hear stuff happening in there, as well as clutter the processing chat with their own. Processing works much better as a closed off room, so that the only conversation people can hear inside of processing is from processing.

**Comment by SkeletalElite** [comment]
Minor thing I noticed
The fluffy pink handcuffs are not normally encased in reinforced walls but can be gotten to via a normal wall if I remember correctly

**Comment by Anticept** [comment]
@deep-fried-faceplant there are shutters for processing. Look at the first post about it.

**Comment by Vivalas** [comment]
This is pretty cool, I like how you can see all cells from the lobby desk.

With the addition of another warden spawn marker maybe adding a second warden job slot would be useful in another PR.

File: _maps/map_files/cyberiad/z2.dmm [hunk 1]
```diff
@@ -499,7 +499,7 @@
...
-"jH" = (/obj/structure/rack,/obj/item/weapon/implanter/adrenalin{pixel_x = 12; pixel_y = -12},/obj/item/weapon/implanter/emp{pixel_x = 9; pixel_y = -9},/obj/item/weapon/implanter/explosive{pixel_x = 6; pixel_y = -6},/obj/item/weapon/implanter/explosive_macro{pixel_x = 3; pixel_y = -3},/obj/item/weapon/implanter/freedom,/obj/item/weapon/implanter/krav_maga{pixel_x = -9; pixel_y = 9},/obj/item/weapon/implanter/loyalty{pixel_x = -6; pixel_y = 6},/obj/item/weapon/implanter/storage{pixel_x = -3; pixel_y = 3},/turf/unsimulated/floor{tag = "icon-floor"; icon_state = "floor"},/area/admin)
+"jH" = (/obj/structure/rack,/obj/item/weapon/implanter/adrenalin{pixel_x = 12; pixel_y = -12},/obj/item/weapon/implanter/emp{pixel_x = 9; pixel_y = -9},/obj/item/weapon/implanter/explosive{pixel_x = 6; pixel_y = -6},/obj/item/weapon/implanter/explosive_macro{pixel_x = 3; pixel_y = -3},/obj/item/weapon/implanter/freedom,/obj/item/weapon/implanter/krav_maga{pixel_x = -9; pixel_y = 9},/obj/item/weapon/implanter/mindshield{pixel_x = -6; pixel_y = 6},/obj/item/weapon/implanter/storage{pixel_x = -3; pixel_y = 3},/turf/unsimulated/floor{tag = "icon-floor"; icon_state = "floor"},/area/admin)
```
Comment (SamHPurp) [inline _maps/map_files/cyberiad/z2.dmm]: Wow @FreeStylaLT . Very nice of you.

Comment (FreeStylaLT) [inline _maps/map_files/cyberiad/z2.dmm]: :dagger:

**Comment by MischievousGremlin** [comment]
Get my fuzzy pink handcuffs out from behind that reinforced wall!

**Comment by Fruerlund** [comment]
Take this area for example, you could retouch it and move the closets around. Remove 2, and move 1 south, and you would have cleared the west closets. 

Add a morgue tray, and a autopsy table and you would have a place where the coroner can bring bodies of a crime after he has made sure they have been dealt with in Medbay. 

This would introduce more to do for both the Detective and the coroner and allow for more cross department collabration. 

A minor touch, easy to edit which could prove to add more depth to the Detective gameplay. 

How would we solve the coroner having access? 

Add a door at the top west side lf evidence, and give the coroner access to security front doors and evidence locker.
http://i.imgur.com/3DMQFCC.jpg

**Comment by Spartan6** [comment]
It's going stale M'lady

**Comment by SamHPurp** [comment]
Notes from me:

1. Sec Request console is on glass, this should be placed on a real wall, not glass. You can't place on glass IC.

2. The prison cells intercoms are also on the glass... could these be pixel edited onto a corner tile instead? 

3. The Mulebot Delivery spot being in Processing feels weird and out of place. Could deliver to the reception desk, or the blank spot left of Warden's door?

4. The exterior warning signs on the Armoury grilles exterior, should be affixed to something that isn't the floor. A grill? a wall?

5. Worth pixel editing the sec toilet (not-perma) so that its /flush/ (honk) up against the wall. Otherwise there is a big gap behind the cistern.

6. Court-room: I get that symmetry is nice, but the huge flag directly in front of a door looks odd.

**Comment by Crazylemon64** [comment]
There's some weird hokery regarding the areas in the execution chamber; patch that up and we can move ahead

**Comment by SamHPurp** [comment]
😱

**Comment by FreeStylaLT** [comment]
@SamHPurp img
@Crazylemon64 Addressed, mini-hallway leading into the chamber is Firing range, the chamber and the observation area are now execution room only.

**Comment by Crazylemon64** [comment]
Here we go!

**Comment by Denthamos** [comment]
I like the more open design, but not how the brig doctor has been squeezed into an even smaller room now.  Will make the job even more annoying to deal with when multiple people are moving around in it when things get busy.

**Pull request merged**